libs/webb-ui-components/src/components/ConnectWalletMobileButton/types.ts

```diff
@@ -1,9 +1,15 @@
 import { ButtonProps } from '../buttons/types';
 import { IWebbComponentBase } from '../../types';
 
+export type MobileButtonProps = {
+  title?: string;
+  extraActionButtons?: Array<ButtonProps>;
+};
+
 export interface ConnectWalletMobileButtonProps
   extends IWebbComponentBase,
     Pick<
       ButtonProps,
       'variant' | 'isFullWidth' | 'leftIcon' | 'rightIcon' | 'size' | 'variant'
-    > {}
+    >,
+    MobileButtonProps {}
```

Comment:
Why don't we just add the new properties inside the interface curly brackets `{}`? 🤔 Or why do you want to add a the `MobileButtonProps` type here.